src/Orchard.Web/Modules/Orchard.Recipes/RecipeHandlers/CommandRecipeHandler.cs

```diff
@@ -13,9 +13,11 @@ namespace Orchard.Recipes.RecipeHandlers  {
     public class CommandRecipeHandler : IRecipeHandler {
         private readonly ICommandManager _commandManager;
         private readonly CommandParser _commandParser;
+        private readonly IRecipeJournal _recipeJournal;
 
-        public CommandRecipeHandler (ICommandManager commandManager) {
+        public CommandRecipeHandler(ICommandManager commandManager, IRecipeJournal recipeJournal) {
             _commandManager = commandManager;
+            _recipeJournal = recipeJournal;

```

Comment:
I do not believe it is necessary as it is always instantiated and no other parameter is checked. Open to suggestion from collaborators.